cli/README.md

```diff
@@ -110,6 +110,7 @@ When using the CLI for the first time there are a few common steps you might wan
 - [`joystream-cli content:deleteVideo`](#joystream-cli-contentdeletevideo)
 - [`joystream-cli content:deleteVideoAsModerator`](#joystream-cli-contentdeletevideoasmoderator)
 - [`joystream-cli content:deleteVideoAssetsAsModerator`](#joystream-cli-contentdeletevideoassetsasmoderator)
+- [`joystream-cli content:getPayoutsOnchainCommitment`](#joystream-cli-contentgetpayoutsonchaincommitment)
```

Comment:
@zeeshanakram3 It seems more commands should be documented here? It misses generating the payload file, etc

Reply:
Yor are right. I have updated the docs

Reply:
Hmm, it seems that some formatting got updated but the README is still missing documentation on new commands? Only `content:getPayoutsOnchainCommitment` is available, shouldn't README also include commands for generating the payload or sending the payouts proposal?